lms/djangoapps/django_comment_client/management/commands/tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+'''
+Tests for management command line tools
+'''
+__author__ = 'azimpradhan'
```

Comment:
makes sense, I've taken it out. I think PyCharm automatically put in the __author__ metadata for me. I've taken it out.